ddp-workspace/projects/ddp-dsm-ui/src/app/filter-column/filter-column.component.ts

```diff
@@ -87,4 +87,8 @@ export class FilterColumnComponent implements OnInit {
     this.dataFilter.notEmpty = false;
     this.dataFilter.empty = false;
   }
+
+  hideCheckbox(dataFilter: Filter): boolean {
```

Comment:
Are there any other contextual clues in the data that we can use to minimize the chance that some other thing called `r21` will be impacted by this?  Perhaps a check somewhere on the study so that this is only applied for RGP?

Reply:
<img width="626" alt="Screenshot 2023-08-14 at 3 27 45 PM" src="https://github.com/broadinstitute/ddp-angular/assets/35533885/6586d229-dc73-49f3-9d2c-d4aa67d5cd79">

^ screenshot of dataFilter. 

@pegahtah and I actually talkd about this. column name `r21` is special and unique to RGP study group. This change will not affect any other studies or columns.